logger/src/lib.rs

```diff
@@ -56,6 +56,17 @@ pub fn init(configuration: &Configuration) -> Result<Option<Telemetries>> {
     Ok(Some(setup_logger(configuration)?))
 }
 
+/// Disables the logger by setting `LOGGER_SET` to true. Will fail
+/// if the logger has already been initialized. This function is
+/// required in order to generate flamegraphs and flamecharts.
+///
+/// Returns true on success.
+pub fn disable_logger() -> bool {
+    !LOGGER_SET
```

Comment:
This is a double negative. Better check if 
```suggestion
    LOGGER_SET.compare_exchange(false, true, ordering::AcqRel, Ordering::Relaxed).is_ok()
```